settings/tests/testthat/test.read.settings.R

```diff
@@ -204,3 +204,10 @@ test_that("check settings runs with only model$name and no database", {
   s1 <- check.settings(update.settings(s))
   expect_identical(s$model$name, s1$model$type)
 })
+
+test_that("invalid pathname is placed in home directory",{
```

Comment:
Then why does the test pass / could you fix the test to check for the intended behavior?